ranges/subset.js

```diff
@@ -68,6 +68,9 @@ const subset = (sub, dom, options = {}) => {
   return true
 }
 
+const minimumVersionWithPreRelease = Object.freeze([new Comparator('>=0.0.0-0')])
+const minimumVersion = Object.freeze([new Comparator('>=0.0.0')])
```

Comment:
this freezes the array, but not the comparator objects - is that an issue?

Reply:
The idea of freezing the array is just to developers do not forget and try to modify it.

Reply:
This feels like an unnecessary guard tbh.

Reply:
is there a measurable difference in memory again? if not, then I'd also say drop it

Reply:
Freeze itself don't save memory, just prevent other to modify.

But I will remove it since don't have too much reason to keep it.